core/domain/services/graphql/resolvers/FieldResolver.php

```diff
@@ -136,6 +138,25 @@ public function resolveId($source)
     }
 
 
+    /**
+     * Resolve the cache ID
+     *
+     * @param mixed     $source
+     * @return string
+     * @since $VID:$
+     */
+    public function resolveCacheId($source)
+    {
+        $model_name = $this->model->item_name();
+        $ID         = $source->ID();
+        $uniqid     = uniqid();
```

Comment:
was about to ask why not `uniqueID` ?
but checked docs and it turns out this is PHP function name
https://www.php.net/manual/en/function.uniqid.php
:smiley: